src/modules/dashboard/sagas/colonyCreate.ts

```diff
@@ -413,59 +406,11 @@ function* colonyCreate({
       );
     }
 
-    /*
-     * Deploy OldRoles
-     */
-    yield put(transactionReady(deployOldRoles.id));
-
-    const {
-      payload: {
-        transaction: {
-          receipt: {
-            logs: [deployOldRolesLog],
-          },
-        },
-      },
-    } = yield takeFrom(
-      deployOldRoles.channel,
-      ActionTypes.TRANSACTION_SUCCEEDED,
-    );
-    const oldRolesAddress = parseExtensionDeployedLog(deployOldRolesLog);
-
-    /*
-     * Set OldRoles role
-     */
-    yield put(
-      transactionAddParams(setOldRolesRole.id, {
-        address: oldRolesAddress,
-      }),
-    );
-    yield put(transactionReady(setOldRolesRole.id));
-    yield takeFrom(setOldRolesRole.channel, ActionTypes.TRANSACTION_SUCCEEDED);
-
     /*
      * Deploy OneTx
      */
     yield put(transactionReady(deployOneTx.id));
 
-    const {
-      payload: {
-        transaction: {
-          receipt: {
-            logs: [deployOneTxLog],
-          },
-        },
-      },
-    } = yield takeFrom(deployOneTx.channel, ActionTypes.TRANSACTION_SUCCEEDED);
-    const oneTxAddress = parseExtensionDeployedLog(deployOneTxLog);
-
-    /*
-     * Set OneTx role
-     */
-    yield put(transactionAddParams(setOneTxRole.id, { address: oneTxAddress }));
```

Comment:
We need to do this `transactionAddParams` for both of the new role setting transactions, so this line should be replaced by two (one for each role that's set). The lines below this should be replaced with the same thing for each new role setting transaction.